scanner/datastore/postgres/distributions.go

```diff
@@ -0,0 +1,39 @@
+package postgres
+
+import (
+	"context"
+
+	"github.com/quay/claircore"
+)
+
+func (m *matcherStore) Distributions(ctx context.Context) ([]claircore.Distribution, error) {
+	const selectDists = `SELECT dist_id, dist_version_id, dist_version FROM vuln GROUP BY dist_id, dist_version_id, dist_version`
```

Comment:
I wonder if `SELECT DISTINCT dist_id, dist_version_id, dist_version` makes more sense since there's no aggregate function used in this `group by`

Reply:
I think either one is about the same, but I'll switch to `DISTINCT` to simplify the query